src/main/java/com/keniu/dto/CreateBookRequestDto.java

```diff
@@ -27,4 +28,5 @@ public class CreateBookRequestDto {
     private String description;
     @Size(max = 255)
     private String coverImage;
+    private Set<Long> categoryIds;
```

Comment:
Do we need validate categoryIds?

Reply:
What do you suggest?

Reply:
You can use `@NotEmpty` and `@Positive` for Long

Reply:
book may not have category, so I think it's better to leave it as it is